src/ci/agent.sh

```diff
@@ -26,3 +26,7 @@ cargo test --release --manifest-path ./onefuzz/Cargo.toml
 
 cp target/release/onefuzz-agent* ../../artifacts/agent
 cp target/release/onefuzz-supervisor* ../../artifacts/agent
+
+if [ -f target/release/onefuzz_agent.pdb ]; then
```

Comment:
Why are we using this as a condition, specifically?

Reply:
(IOW, why not use `onefuzz_supervisor.pdb`, or unconditionally do `cp target/release/onefuzz*.pdb [...]` or similar?)

Reply:
Updated to use for loop with globbing